solidity/contracts/token/HypNative.sol

```diff
@@ -33,8 +39,9 @@ contract HypNative is FungibleTokenRouter {
         address _hook,
         address _interchainSecurityModule,
         address _owner
-    ) public initializer {
+    ) public virtual reinitializer(2) {
         _MailboxClient_initialize(_hook, _interchainSecurityModule, _owner);
+        _MovableCollateralRouter_initialize(_owner);
```

Comment:
can we add a comment about the initialized version?

Reply:
Fixed